src/components/admin/activities/activity-type-select.js

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import {Select, InputLabel, FormControl, MenuItem } from "@material-ui/core";
+import { dummyData } from './activities-const'
+import { withStyles } from '@material-ui/styles';
+
+
+const useStyles = theme => ({
+    activityselect: {
```

Comment:
Instead of using styles, you can get the same result by using `Grid` for layouting.